polyply/tests/test_lib_files.py

```diff
@@ -85,7 +85,8 @@ def assert_equal_blocks(block1, block2):
                     if _interaction_equal(ref_inter, new_inter, inter_type):
                         break
                 else:
-                    return False
+                    assert False
+        assert True
```

Comment:
This is a void statement and should be removed

Reply:
indeed works without ;)